### PR TITLE
Set a auth_uri in glance-api.conf

### PR DESCRIPTION
**Pull request by dirkmueller**
Silences a warning on glance startup. Not sure if it
really has any real effect.

### CODE REVIEW

**Comment by vuntz** [comment]
Wasn't this deprecated? @toabctl?

**Comment by dirkmueller** [comment]
no, the other auth_ settings are deprecated in favor of auth_uri (which provides one setting for everything that was specified as individual components beforehand)

**Comment by toabctl** [comment]
Oh. Sorry. Forgot to answer. @dirkmueller I think that's not correct. ```auth_``` are deprecated in favor of ```identity_uri``` but ```auth_uri``` is still valid. ```auth_uri``` is the complete public identity endpoint and ```identity_uri``` is the complete admin endpoint.

**Comment by vuntz** [comment]
@toabctl so I guess you're saying it doesn't hurt to have both?

**Comment by vuntz** [comment]
+1

File: chef/cookbooks/glance/templates/default/glance-api.conf.erb [hunk 1]
```diff
@@ -387,6 +387,7 @@ identity_uri = <%= @keystone_settings['protocol'] %>://<%= @keystone_settings['i
 admin_tenant_name = <%= @keystone_settings['service_tenant'] %>
```
Comment (vuntz) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: Btw, why don't we use `<%= @keystone_settings['admin_auth_url'] %>` for `identity_uri` at https://github.com/crowbar/barclamp-glance/pull/252/files#diff-86b6a6b811558c57ef953fa962833ccaR386 ?

Comment (dirkmueller) [inline chef/cookbooks/glance/templates/default/glance-api.conf.erb]: Thats broken, fixed in a separate commit.

**Comment by vuntz** [comment]
+1

**Comment by toabctl** [comment]
> @toabctl so I guess you're saying it doesn't hurt to have both?

@vuntz: I would say you **need** both (both == ```identity_uri``` and ```auth_uri```).

**Comment by toabctl** [comment]
+1

**Pull request merged**